src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IBinaryOperatorExpression.vb

```diff
@@ -479,18 +479,18 @@ IBlockStatement (26 statements, 3 locals) (OperationKind.BlockStatement) (Syntax
             Operand: 
               IBinaryOperatorExpression (BinaryOperatorKind.Concatenate, Checked) (OperationKind.BinaryOperatorExpression, Type: System.String) (Syntax: 'x & y')
                 Left: 
-                  IConversionExpression (Explicit, TryCast: False, Unchecked) (OperationKind.ConversionExpression, Type: System.String) (Syntax: 'x')
+                  IConversionExpression (Implicit, TryCast: False, Unchecked) (OperationKind.ConversionExpression, Type: System.String, IsImplicit) (Syntax: 'x')
```

Comment:
Is this a consequence of the `ISimpleAssignmentExpression` being `Implicit`? i.e., `Implicit`ness is transitive in some sense?

That seems fine... but I'm confused as to why this was classified as an explicit conversion to begin with. Where's the explicit conversion in `x & y`?

Reply:
> Is this a consequence of the ISimpleAssignmentExpression being Implicit? i.e., Implicitness is transitive in some sense?

No, this isn't related to ISimpleAssignmentExpression node at all. Implicit conversions within concatenate operator aren't supposed to produce any diagnostics originated from the fact that they are implicit. And that was leaking to the IConversionExpression node, see the logic around ```explicitSemanticForConcatArgument``` in Binder_Operators.vb.